src/pat/inject.js

```diff
@@ -112,6 +112,24 @@ define([
             _.execute(cfgs, $el);
         },
 
+        onButtonClick: function inject_onButtonClick(ev) {
+            ajax.onClickSubmit(ev); // make sure the submitting button is sent with the form
```

Comment:
This ``onClickSubmit`` method in the ajax.js pattern is actually a misnomer. It makes it sound as if it submits the form, but instead it merely sets data on the form element.

We should consider renaming it to something like ``setFormData``.